src/Contracts/ConstraintFinderInterface.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Db\Contracts;
+
+use Yiisoft\Db\Constraints\Constraint;
+use Yiisoft\Db\Constraints\CheckConstraint;
+use Yiisoft\Db\Constraints\DefaultValueConstraint;
+use Yiisoft\Db\Constraints\ForeignKeyConstraint;
+use Yiisoft\Db\Constraints\IndexConstraint;
+
+/**
+ * ConstraintFinderInterface defines methods for getting a table constraint information.
+ */
+interface ConstraintFinderInterface
```

Comment:
What is the purpose for introducing the interface? Are we going to change schema to another implementation? If so, the interface should be somehow used. It's not used currently.